server/utils/bookingUtils.ts

```diff
@@ -104,10 +104,15 @@ export const allStatuses: Array<{ name: string; id: Booking['status']; tagClass:
     tagClass: 'govuk-tag--green',
   },
   {
-    name: 'Departed',
+    name: config.flags.turnaroundsDisabled ? 'Departed' : 'Departed - Turnaround In Progress',
```

Comment:
I ran this by Selina (cos capitalisation is a bugbear of mine) but she says 'Departed - turnaround in progress' should now just say 'Turnaround'

Reply:
Will chase - this has knock-on UI implications for the booking search